test/unit/comment_test.rb

```diff
@@ -308,4 +308,12 @@ class CommentTest < ActiveSupport::TestCase
     assert_not_nil graph
     assert graph.class, Hash
   end
+
+  test 'find email using twitter user name' do
+    require 'yaml'
```

Comment:
I don't think you need this line...

Reply:
@siaw23 Actually I looked at few other tests in the file like
```
  test 'should have gmail quote' do
    require 'mail'
    require 'nokogiri'
    mail = Mail.read('test/fixtures/incoming_test_emails/gmail/incoming_gmail_email.eml') 
    mail_doc = Nokogiri::HTML(mail.html_part.body.decoded) # To parse the mail to extract comment content and reply content 
    gmail_quote = Comment.gmail_quote_present?(mail_doc)
    assert_equal gmail_quote, true
  end
```
So taking inspiration from these, I had used this way.
So do I really need to hardcode the username?